src/app/authentication/pages/login/login.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
```

Comment:
@carryall Should we remove this extra line?

Reply:
@Lahphim I'm not sure because this file is generated with `ng g component <component-name>` with an extra line at the beginning and the end of the class 🤔  if we want to remove it every time we generate a new component we have to manually remove it.

Reply:
@Lahphim @junan removed in [e7e6610](https://github.com/carryall/survey-challenge/pull/23/commits/e7e661025ee4adc009d33c979a1c8e00468638d8)